src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Square extends Default {
```

Comment:
The class name 'Square' is appropriate as it is a noun and represents a shape. However, ensure that the class 'Default' is correctly used and named, as it might not be descriptive enough.